src/github.com/getlantern/testify/.gitignore

```diff
@@ -0,0 +1,24 @@
+# Compiled Object files, Static and Dynamic libs (Shared Objects)
```

Comment:
Unfortunately, measured and http-proxy still have some tests that rely on getlantern/testify.  I'll fix that and then remove this.